cluster-healthcheck.metrics.md

```diff
@@ -0,0 +1,108 @@
+# The 12 most important cluster monitoring points
+
+1. [Broker count](#broker-count)
+2. [Active Controller count](#active-controller-count)
+3. [Offline partitions](#offline-partitions)
+4. [Unclean Elections](#unclean-elections)
+5. [Under Replicated Partitions](#under-replicated-partitions)
+6. [Under Min In Sync Replicas](#under-min-in-sync-replicas)
+7. [Max lag by groupId by partition](#max-lag-by-groupid-by-partition)
+8. [Broker IO Activity](#broker-io-activity)
+9. [Broker Net Activity](#broker-network-activity)
+9. [Zookeper Avg Latency](#zookeeper-avg-latency)
+10. [Zookeper Connections](#zookeeper-connections)
+11. [Broker Zookeeper disconnections](#broker-zookeeper-disconnections)
+
+## Broker count
+* **Type**: Cluster Health/Message Delivery
+* **Description**: The number of Kafka broker in you cluster has a direct impact on the business reliability you want to guarantee. When `min.isr` set and `count of broker < min.isr`, the the data production is stopped and the producers start a retry period before timing out or worse reach the limits of the hosting machine.
+* **Metric**: When collecting broker metrics, get the host cardinality.
+* **Notification**: Alarm when `count of broker < min(producer min.isr)`
```

Comment:
Not sure about what we track here, how should we get access to this ?
Do we collect it with JMX ? by checking that processus are started ? By checking the BrokerState in JMX : `kafka.server:type=KafkaServer,name=BrokerState` ?

The description seems redundant with other checks that already tracks under replicated/min.isr partitions.

Reply:
+1
The broker count is an interesting information but not a health check aspect. Depending on your focus (speed vs durability) you may not be interested at all by this information.
I propose to remove the paragraph and give priority to the under min isr or under replicated sections because they are critical in a cluster health check.